src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+.img-responsive {
+  width: 100%;
+}
+.list-none {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+.word-break{
+  word-break: break-word;

```

Comment:
this property is now officially deprecated and we not gonna use it anymore from now. Instead of this use overflow-wrap